src/api-v1.js

```diff
@@ -151,9 +151,83 @@ async function validateWorkerType (ctx, workerTypeName, workerType) {
 
 }
 
+/**
+ * Calculate some summary statistics for a worker type, based on the given
+ * WorkerState.
+ */
+function workerTypeSummary(workerType, workerState) {
+  let summary = {
+    workerType: workerType.workerType,
+    minCapacity: workerType.minCapacity,
+    maxCapacity: workerType.maxCapacity,
+    requestedCapacity: 0,
+    pendingCapacity: 0,
+    runningCapacity: 0,
+  };
+
+  let capacities = {};
+  workerType.instanceTypes.forEach(instanceType => {
+    capacities[instanceType.instanceType] = instanceType.capacity;
+  });
+
+  workerState.instances.forEach(instance => {
+    if (instance.state === 'running') {
+      summary.runningCapacity += capacities[instance.type] || 0;
+    } else if (instance.state === 'pending') {
+      summary.pendingCapacity += capacities[instance.type] || 0;
+    } // note that other states are ignored
+  });
+
+  workerState.requests.forEach(request => {
+    summary.requestedCapacity += capacities[request.type] || 0;
+  });
+
+  return summary;
+}
+
 module.exports = api;
 
 api.declare({
+  method: 'get',
+  route: '/worker-types',
```

Comment:
Actually, I'm making this consistent with other services and REST principals, so I'd prefer to keep `/worker-types`.  I could be convinced to use `/worker-type` since the rest of the endpoints are also singular, but as jonas suggested we may want to change those some day, too.